site/content/docs/main/upgrade-to-1.11.md

```diff
@@ -76,22 +77,41 @@ Before upgrading, check the [Velero compatibility matrix](https://github.com/vmw
 
     ```bash
     Client:
-        Version: v1.10.0
+        Version: v1.11.0
         Git commit: <git SHA>
 
     Server:
-        Version: v1.10.0
+        Version: v1.11.0
     ```
+
+### Upgrade from v1.10
+If it's directly upgraded from v1.10, the other steps remain the same only except for step 3 above. The details as below:
+
+3. Update the container image used by the Velero deployment, plugin and, optionally, the node agent daemon set:
+    ```bash
+   # set the container and image of the init container for plugin accordingly,
+   # if you are using other plugin
+    kubectl set image deployment/velero \
+        velero=velero/velero:v1.11.0 \
+        velero-plugin-for-aws=velero/velero-plugin-for-aws:v1.6.1 \
```

Comment:
nit: v1.6.1 plugin is for velero v1.10.x, here it should be v1.7.0

Reply:
updated